dist/chart-legend/chart-legend.css

```diff
@@ -0,0 +1,103 @@
+.chart-legend {
+  display: inline-block;
+}
+.chart-legend [hidden] {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
+.chart-legend.table {
+  flex-shrink: 0;
+}
+.chart-legend ul {
+  list-style: none;
+  margin: 0;
+  -webkit-padding-start: 24px;
+          padding-inline-start: 24px;
+}
+.chart-legend ul li {
+  line-height: 2rem;
+  position: relative;
+  transition: opacity 0.5s;
+}
+.chart-legend ul li.primary::before {
```

Comment:
To satisfy BEM, these selectors would need to read something like:

`.chart-legend ul li.chart-legend__item--primary::before`

Which, if you don't need the specificity, can most likely be reduced to:

`.chart-legend__item--primary::before`